assignment2/assign2_CRUD.c

```diff
@@ -0,0 +1,193 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+
+#define FILENAME "users.txt"
+#define MAX_LEN 100
+
+typedef struct {
+    int id;
+    char name[MAX_LEN];
+    int age;
+} User;
+
+void createFile();
+void addUser();
+void display();
+void update();
```

Comment:
Update code so methods like update or delete should send a result for success or failure (int)